menu_student.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base_student.html' %}
+{% block body %}
+
+<table><tr><td width=150px; max-width=150px; valign='top'; style="border-right: 1px solid black; padding: 5px;">
+  <h3>Step 1</h3>
+  <li><a href="/preregistration?{{ session_query }}">View My Catalog</a>
+  <li><a href="/catalog_full?{{ session_query }}">View Full Catalog</a>
+  <h3>Step 2</h3>
+  <li><a href="/schedule?{{ session_query }}">Register For Classes</a>
+  <li><a href="/preferences?{{ session_query }}">Order By Preference</a>
+  <h3>Step 3</h3>
+  <li><a href="/postregistration?{{ session_query }}">View Final Schedule</a>
+</td>
```

Comment:
If some steps are not available to the student yet, we can put the "Comming Soon" message right here and remove the link. This can be done with a jinja "if" statement. They shouldn't have to click a link to discover that they don't have access yet.